elements/noflo-graph-inspector.html

```diff
@@ -272,18 +284,18 @@ <h1 on-keydown="checkUpdateName" on-blur="updateName" contenteditable="">{{graph
           return;
         }
         this.htmlEditor = CodeMirror(this.$.html_editor, {
-          lineNumbers: true,
+          lineNumbers: false,
```

Comment:
@bergie why lline numbers are disabled? This seems unrelated

Reply:
It seemed to trigger a rendering bug with CodeMirror when used in a narrow space

With line numbering:

![screenshot 2017-11-13 at 21 11 30](https://user-images.githubusercontent.com/3346/32746870-4905bb6e-c8b7-11e7-92a6-fc66e4ed8f2e.png)

Without:

![screenshot 2017-11-13 at 21 11 41](https://user-images.githubusercontent.com/3346/32746883-4edf5950-c8b7-11e7-967d-142699e18fa8.png)

Reply:
Editing component code without line numbers is a bit shit...

Reply:
This is not for component code, only for graph preview HTML (and graph specs)